src/components/ui/PrismCode.tsx

```diff
@@ -1,28 +1,46 @@
 import React from 'react';
 import Highlight, { defaultProps, Language, PrismTheme } from 'prism-react-renderer';
-import { Text, TextVariants } from '@patternfly/react-core';
+import { Text, TextVariants, ClipboardCopy, clipboardCopyFunc } from '@patternfly/react-core';
 import defaultTheme from 'prism-react-renderer/themes/github';
 import './PrismCode.css';
-
 type PrismCodeProps = {
   code: string;
   language?: Language;
   theme?: PrismTheme;
+  copiable?: boolean;
 };
-
-const PrismCode: React.FC<PrismCodeProps> = ({ code, language = 'bash', theme = defaultTheme }) => (
+const PrismCode: React.FC<PrismCodeProps> = ({
+  code,
+  language = 'bash',
+  theme = defaultTheme,
+  copiable = false,
+}) => (
   <Highlight {...defaultProps} code={code} language={language} theme={theme}>
-    {({ className, style, tokens, getLineProps, getTokenProps }) => (
-      <Text component={TextVariants.pre} className={className} style={style}>
-        {tokens.map((line, i) => (
-          <div key={i} {...getLineProps({ line, key: i })}>
-            {line.map((token, key) => (
-              <span key={key} {...getTokenProps({ token, key })} />
+    {({ className, style, tokens, getLineProps, getTokenProps }) => {
```

Comment:
nitpick:
```suggestion
    {({ className, style, tokens, getLineProps, getTokenProps }) => (<>FOO</>)
```

(we can skip `return` in the arrow function)

Reply:
Done.